app/services/adapters/runner_callback.rb

```diff
@@ -0,0 +1,27 @@
+module Adapters
+  class RunnerCallback
+
+    class FrontendRequestError < StandardError
+    end
```

Comment:
Given the namespacing, wonder if we can rename this to something more simple:
Adapters:: RunnerCallback::ClientRequestError